samples/subsys/display/lvgl/boards/mini_stm32h743.conf

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Charles Dias <charlesdias.cd@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable SPI
+CONFIG_SPI=y
```

Comment:
According to https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/display/Kconfig.st7735r#L10, this line is not required.

Reply:
Removed.